### PR TITLE
Add data-testid attribute for e2e tests

### PR DESCRIPTION
**Pull request by vitorflg**
## What's the purpose of this pull request?

It adds a `data-testid` attribute to the main elements of its repository packages that permit storefronts to do end-end tests.

## How it works / How to test it?? 
The description of this [PR](https://github.com/vtex-sites/marinbrasil.store/pull/350/files) explains how to run the tests inside a theme.

## References
X

### CODE REVIEW

**Review by Gmantiqueira — APPROVED** [review]

**Review by tlgimenes — APPROVED** [review]

File: docs/e2e-testing.md [hunk 1]
```diff
@@ -54,39 +54,39 @@ In the following section, you'll learn how to build your own E2E test with Cypre
    │   ├── plugins
    ```
 
-Now you're ready to start developing your own E2E tests! Additionally, you can install the VTEX helper functions hub by running (command - WIP) to make writing E2E tests easier.
+Now you're ready to start developing your own E2E tests! Additionally, you can install the [VTEX helper functions hub](https://github.com/vtex/test-toolshttps://github.com/vtex/test-tools) to make writing E2E tests easier.
```
Comment (igorbrasileiro) [inline docs/e2e-testing.md:57]: @vitorflg this is wrong, right?
```suggestion
Now you're ready to start developing your own E2E tests! Additionally, you can install the [VTEX helper functions hub](https://github.com/vtex/test-tools) to make writing E2E tests easier.
```

File: docs/e2e-testing.md [hunk 2]
```diff
@@ -54,39 +54,39 @@ In the following section, you'll learn how to build your own E2E test with Cypre
...
 
 ### Step 2: Writing a sample test
 
 In the following, we'll create a simple test that verifies how a specific product search behaves. Use it as an example to create your own E2E test with Cypress.
 
 1. Open up your SFJ project in the code editor of your choosing.
 2. Create a new file named `index.js` inside `cypress/e2e/{test-name}` (e.g., `cypress/e2e/search`).
-3. Using JavaScript, write your E2E test. Take the code below as an example:
+3. Make sure to add the `data-testid` attribute to the elements that will need some interaction. In our case are `searchBarInput`, `searchBarItem`, and `productTitle`.
+4. Using JavaScript, write your E2E test. Take the code below as an example:
```
Comment (igorbrasileiro) [inline docs/e2e-testing.md:66]: Why we don't recommend using TS?

**Review by igorbrasileiro — APPROVED** [review]
LGTM, left 2 comments, take a look there.

File: docs/e2e-testing.md (hunk 2 above)
Comment (Gmantiqueira) [reply docs/e2e-testing.md:66]: I think it doesn't matter if it's JS or TS. It's like "use JS in anyhow". [There's support to TS](https://docs.cypress.io/guides/tooling/typescript-support) anyway.

File: packages/store-ui/src/SearchFilter/AccordionCollapsibleIcon.tsx [hunk 3]
```diff
@@ -25,7 +25,10 @@ export const SearchFilterAccordionCollapsibleIcon: FC<Props> = ({
   isActive,
   variant,
 }) => (
-  <Box variant={`${variant}.icon`}>
+  <Box
+    data-testid="searchFilterAccordionCollaipsibleIcon"
```
Comment (Gmantiqueira) [inline packages/store-ui/src/SearchFilter/AccordionCollapsibleIcon.tsx:29]: typo: `Collaipsible` to `Collapsible`

**Comment by emersonlaurentino** [comment]
@bentoper can u rebase with `master`?

**Review by tlgimenes — APPROVED** [review]

**Comment by tlgimenes** [comment]
Please rebase

**Review by emersonlaurentino — APPROVED** [review]

**Pull request merged**